Cabal/Distribution/Types/ForeignLib.hs

```diff
@@ -42,16 +53,81 @@ data ForeignLib = ForeignLib {
     }
     deriving (Generic, Show, Read, Eq, Typeable, Data)
 
+data LibVersionInfo = LibVersionInfo Int Int Int deriving (Data, Eq, Generic, Typeable)
```

Comment:
I will refer to libtool more clearly here.
I can also add these accessors. There is also `libVersionInfoCRA` which gives you a tuple of the three numbers, but I can see how that is not sufficient and/or clear enough.